bai-tap-buoi-9/main.js

```diff
@@ -1 +1,151 @@
-const 
\ No newline at end of file
+// Hàm kiểm tra số nguyên
+function checkNumber(number) {
+    if (parseInt(number)==number) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+// Bài 1:
+const realSalary = (money) => {
+    if (money >= 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.3)
+        console.log('Lương ròng: '+money*0.7)
+    }
+    else if (money >= 7000000 && money < 15000000) {
+        console.log('Thuế thu nhập: ' + money*0.2)
+        console.log('Lương ròng: '+money*0.8)
+    }
+    else if (money >= 0 && money < 7000000) {
+        console.log('Thuế thu nhập: ' + money*0.1)
+        console.log('Lương ròng: '+money*0.9)
```

Comment:
```suggestion
      console.log('Thuế thu nhập: ' + money * 0.1)
      console.log('Lương ròng: ' + money * 0.9)
```
Chú ý các dấu cách và indent